kitsune/sumo/static/sumo/js/wiki.editor.js

```diff
@@ -173,34 +162,49 @@ function clearSelected(name) {
 // called by the time the following code is run, because it always
 // waits until all deferred scripts have been loaded, and the code
 // in this file is always bundled into a script that is deferred.
-document.addEventListener("DOMContentLoaded", function() {
+document.addEventListener("DOMContentLoaded", function () {
   // Enable/disable products and topics based on initial selections.
   handleUpdate();
   document.querySelectorAll(
     '#relevant-topics input[type="checkbox"][name="topics"]'
-  ).forEach(function(topic) {
-    topic.addEventListener("click", function() {
+  ).forEach(function (topic) {
+    topic.addEventListener("click", function () {
       handleTopicRelationships(topic);
       handleUpdate();
     });
   });
   document.querySelectorAll(
     '#relevant-products input[type="checkbox"][name="products"]'
-  ).forEach(function(product) {
-    product.addEventListener("click", function() {
+  ).forEach(function (product) {
+    product.addEventListener("click", function () {
       handleUpdate();
     });
   });
   document.getElementById(
     "relevant-topics-clear-selected"
-  ).addEventListener("click", function(event) {
+  ).addEventListener("click", function (event) {
     event.preventDefault();
     clearSelected("topics");
   });
   document.getElementById(
     "relevant-products-clear-selected"
-  ).addEventListener("click", function(event) {
+  ).addEventListener("click", function (event) {
     event.preventDefault();
     clearSelected("products");
   });
+
+  new TomSelect("select[id='id_restrict_to_groups']", {
+    closeAfterSelect: true,
+    plugins: {
+      clear_button: {
```

Comment:
@escattone `tom-select` doesn't have an `x` for the clear button but rather a checkbox. 
See https://tom-select.js.org/plugins/clear-button/

We could override this but unless there are strong objections I would prefer to tackle this in a different PR/issue to avoid scope creep. Thoughts?

Reply:
Yes, let's tackle that in another PR/issue!

However, I do see a small `x` both in our HTML and on their website.
<img width="1213" alt="image" src="https://github.com/user-attachments/assets/f819c463-8d9b-4b3a-90e9-68504daff796">

Reply:
that's for Bootstrap 4/5. If you select the `Default` from the drop down it's the checkbox..